### PR TITLE
Poster 

### PR DESCRIPTION
**Pull request by VividPups**
## About the PR
adds 2 posters
## Why / Balance
requested 
## How to test
spawn it 
## Media
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**
:cl:
- add: 1 new posters
-->

### CODE REVIEW

**Comment by Tych0theSynth** [comment]
I'm struggling to understand the relevance of the Uncle Sam poster for Frontier - where does it fit in?

Uncle Sam is an incredibly political figure, having been used in (largely) propaganda since his inception in the 19th century. He is also very controversial for many (within or outside of the US). Personally, the potential inclusion of this in Frontier makes me uncomfortable and I don't think it fits the setting. 

NanoTrasen/Syndicate/other corporate propaganda always makes sense with the setting of the game, but this feels too 'real-world' political to me, which is something I play this game to escape from. 

While I like the design of the Meatzu poster, I'm not sure if we should be advertising individual restaurants?
There are also a LOT of unnecessary capital letters in that description.

**Comment by Cerebral-Error** [comment]
> While I like the design of the Meatzu poster, I'm not sure if we should be advertising individual restaurants? There are also a LOT of unnecessary capital letters in that description.

As the one who made the Meatzu poster, this might have been an error on my part. I thought there was a push for some posters for in-game factions/businesses. While I'm glad you like the design of it, if it's not appropriate to have that level of individual advertisement, we don't need to include it.

**Comment by Tych0theSynth** [comment]
> As the one who made the Meatzu poster, this might have been an error on my part. I thought there was a push for some posters for in-game factions/businesses. While I'm glad you like the design of it, if it's not appropriate to have that level of individual advertisement, we don't need to include it.

At the end of the day, I'm not a maintainer and it isn't up to me. These are just bits of feedback from myself as an individual :)
I think the difference is that while we have posters for shipyards, these are 'static' businesses that don't actually make any money in round, as opposed to a business someone is actually running during a round and making money from.

**Comment by VividPups** [comment]
> I'm struggling to understand the relevance of the Uncle Sam poster for Frontier - where does it fit in?
> 
> Uncle Sam is an incredibly political figure, having been used in (largely) propaganda since his inception in the 19th century. He is also very controversial for many (within or outside of the US). Personally, the potential inclusion of this in Frontier makes me uncomfortable and I don't think it fits the setting.
> 
> NanoTrasen/Syndicate/other corporate propaganda always makes sense with the setting of the game, but this feels too 'real-world' political to me, which is something I play this game to escape from.
> 
> While I like the design of the Meatzu poster, I'm not sure if we should be advertising individual restaurants? There are also a LOT of unnecessary capital letters in that description.

Sam's Gun shops. How does a poster of that has a like 1 in 50 chance of spawning make the game feel top real world politicals especially when the actual game has Terrorists that comes in kill people and the use a nuke

**Comment by VividPups** [comment]
> > While I like the design of the Meatzu poster, I'm not sure if we should be advertising individual restaurants? There are also a LOT of unnecessary capital letters in that description.
> 
> As the one who made the Meatzu poster, this might have been an error on my part. I thought there was a push for some posters for in-game factions/businesses. While I'm glad you like the design of it, if it's not appropriate to have that level of individual advertisement, we don't need to include it.

There isn't a reason why A player made fraction/business isn't allow. Look at McCargo

**Comment by Tych0theSynth** [comment]
The poster is literally just Uncle Sam, which is why I don't think it's appropriate, but unfortunately think you've missed the point I was trying to make (the American flag behind him isn't even accurate, too) :/

What's your point about McCargo?

**Comment by VividPups** [comment]
> The poster is just Uncle Sam, which is why I don't think it's appropriate, but unfortunately think you've missed the point I was trying to make (the American flag behind him isn't even accurate, too) :/
> 
> What's your point about McCargo? It's a ship with a specific theme.

yes I know the flag isn't fully accurate because I am trying to match the other posters. also McCargo was more then just "a ship with a theme" when there were a lot of custom-made things. which is what this poster is going to be with a Custom event coming in the future

**Comment by Tych0theSynth** [comment]
Well, I've shared my thoughts. I don't think it's a good idea, but you do you.

**Comment by arimah** [comment]
On the topic of Meatzu: The issue I take with it is that it's not a generic faction that just exists in universe. McCargo is just a part of Frontier; no single player owns them, even if Darin is the main person who adds more McCargo stuff. Anyone can run a McCargo franchise. Pleased to Meatzu, on the other hand, is the name of a restaurant run by, to my knowledge, exactly one player. It's not an established _franchise_ that exists independently of that one character. That's why it feels a bit strange. In the same way, I would not PR (or approve) a poster for my own character's restaurant, Spacegrown.

**Comment by Cerebral-Error** [comment]
> On the topic of Meatzu: The issue I take with it is that it's not a generic faction that just exists in universe. McCargo is just a part of Frontier; no single player owns them, even if Darin is the main person who adds more McCargo stuff. Anyone can run a McCargo franchise. Pleased to Meatzu, on the other hand, is the name of a restaurant run by, to my knowledge, exactly one player. It's not an established _franchise_ that exists independently of that one character. That's why it feels a bit strange. In the same way, I would not PR (or approve) a poster for my own character's restaurant, Spacegrown.

I'm more than okay with the Meatzu poster being taken out of this.

**Comment by arimah** [comment]
Regarding Sam: It appears rather obviously to be a version of Uncle Sam. Especially with the facsimile of the US flag in the background, how could anyone interpret it any other way? Given the long and controversial history of that character in the real world, I believe it would be an inappropriate injection of real-life politics into Frontier. There is, additionally, no reason to think the USA exists in any meaningful form in the SS14 universe. The _only_ other sign of that nation, to my knowledge, is the US flag bedsheet, which has always felt incredibly out of place to me. I would not be comfortable adding Uncle Sam to Frontier.

**Comment by arimah** [comment]
> I'm more than okay with the Meatzu poster being taken out of this.

For the record, I think the Meatzu poster looks very nice in its elegant minimalism, and I would not at _all_ be opposed to it being rebranded to something neutral or non-player-specific.

**Comment by VividPups** [comment]
> Regarding Sam: It appears rather obviously to be a version of Uncle Sam. Especially with the facsimile of the US flag in the background, how could anyone interpret it any other way? Given the long and controversial history of that character in the real world, I believe it would be an inappropriate injection of real-life politics into Frontier. There is, additionally, no reason to think the USA exists in any meaningful form in the SS14 universe. The _only_ other sign of that nation, to my knowledge, is the US flag bedsheet, which has always felt incredibly out of place to me. I would not be comfortable adding Uncle Sam to Frontier.

again the sticker is already on the discord.  I doubt people are going to really have some deep and meaningful thoughts about a poster that has a 1 in like 50 chance of spawning and they are most likely walk by not to mention the incredible amount of conversable things already in SS14 already. again this poster is going to be apart of a up coming event

**Comment by Cerebral-Error** [comment]
> For the record, I think the Meatzu poster looks very nice in its elegant minimalism, and I would not at _all_ be opposed to it being rebranded to something neutral or non-player-specific.

I'm not completely sure what it could be advertising that's more neutral, but I will try and think of something!

**Comment by arimah** [comment]
> again the sticker is already on the discord. I doubt people are going to really have some deep and meaningful thoughts about a poster that has a 1 in like 50 chance of spawning and they are most likely walk by not to mention the incredible amount of conversable things already in SS14 already.

As you might glean from the comments on this PR so far, none of them touch on the probability of the poster appearing, because that is immaterial. Nor does it matter how established the sticker is on Discord: this PR is for the game, which _is_ meaningfully distinct from the Discord community. Moreover, two people have already expressed thoughts about the poster.

> again this poster is going to be apart of a up coming event

So presumably lots of people would be exposed to it. That seems to contradict your earlier statement that it would be rare. Even if it's _normally_ rare, putting it in people's faces makes sure they have to see it and potentially think about it.

**Comment by VividPups** [comment]
> > again the sticker is already on the discord. I doubt people are going to really have some deep and meaningful thoughts about a poster that has a 1 in like 50 chance of spawning and they are most likely walk by not to mention the incredible amount of conversable things already in SS14 already.
> 
> As you might glean from the comments on this PR so far, none of them touch on the probability of the poster appearing, because that is immaterial. Nor does it matter how established the sticker is on Discord: this PR is for the game, which _is_ meaningfully distinct from the Discord community. Moreover, two people have already expressed thoughts about the poster.
> 
> > again this poster is going to be apart of a up coming event
> 
> So presumably lots of people would be exposed to it. That seems to contradict your earlier statement that it would be rare. Even if it's _normally_ rare, putting it in people's faces makes sure they have to see it and potentially think about it.

with how the event will be, they will not care about the poster at all I'm sure. also yes only 2 people "express thoughts on it"  and no I wouldn't say it is meaningful and distant. I mean there is a liberty station that has a Flag on it.  The Mosin is a soviet Weapon the AKMS is a soviet Weapon, and the Ushanka is the poster that says "Communist party".  the SVT is a Soviet weapon also. there is Clear references to the two nations.

**Comment by arimah** [comment]
> I mean there is a liberty station that has a Flag on it.

Oh yes, of course. I forgot about the Liberation Station. That one has also always struck me as weirdly out of place.

> The Mosin is a soviet Weapon the AKMS is a soviet Weapon

Having been developed in the former Soviet Union does not mean they are intrinsically "Soviet weapons". The Mosin has been manufactured over the last century-and-a-bit by over a dozen countries in various forms, and the AKMS enjoys a similar status.

> the Ushanka is the poster that says "Communist party"

The ushanka does not belong to the USSR; it is a design that exists throughout Central and Eastern Europe, and all over the Balkans. The Community Party poster displays the sickle and hammer motif in USSR colours – if you ask me, a somewhat problematic reference to real-life politics. There is no reason to _add_ to that.

**Comment by VividPups** [comment]
ak47 is in fact a soviet weapon the akms is in fact a soviet weapon regardless of what country made a varient of the origin of the AKMS and AK47 is Soviet Russia made weapon 

You realize the Name of it mosin in game is from a Soviet scientist 

the reason why Soviet stuff was added is because there is a Third Soviet Union Lore in SS14 

the Ushanka Hat has the Soviet Star on it in game.  that's what the Gold and Red dot is.

**Comment by Tych0theSynth** [comment]
> You realize the Name of it mosin in game is from a Soviet scientist

Sergei Mosin wasn't Soviet.

**Comment by VividPups** [comment]
> > You realize the Name of it mosin in game is from a Soviet scientist
> 
> Sergei Mosin wasn't Soviet.

image

image

File: Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml [hunk 1]
```diff
@@ -154,4 +154,6 @@
       - PosterLegitRenault
       - PosterLegitNTTGC
       - NFPosterLegitMagnusWatching # Frontier
+      - NFPosterMeatzu # Frontier
+      - NFPosterSAM
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml:158]: Please mark as legitimate.

**Review by whatston3 — COMMENTED** [review]
Meatzu poster could really use an outline and maybe a bit of internal texture - here's a selection of four posters in-game.  Each has more texture than the Meatzu one, and each has a distinct (if not entirely solid) outline.

image

Descriptions are nice and short - unsure if NT'd want concepts like nationhood as a legitimate poster.  You could turn it into some contraband poster advertising the "Sons of Liberty" or some such thing, promoting establishing a rogue state on ancient American values or something.  Play off the setting. 😃

**Comment by VividPups** [comment]
Can you please convert this into a draft

**Comment by ErhardSteinhauer** [comment]
Closing this PR because it was stale for quite some time now and because of controversy around uncle sam poster. Wouldn't mind seeing metzu poster in game though.

**Comment by VividPups** [comment]
its stale because the people that "had a problem with it" didn't replay and I was waiting for more back and worth. It shouldn't be close because no other maintainer or maptainer said anything.  sooo much controversy over an uncles sam poster but not the Communism Hammer and sickle weapons, symbols and references

**Pull request closed without merging**